### PR TITLE
luigi, pkg: allow parsing JSONs with non-string values

### PR DESCRIPTION
**Pull request by kopiczko**

### CODE REVIEW

File: format_test.go [hunk 1]
```diff
@@ -117,6 +117,12 @@ func Test_format(t *testing.T) {
 			expectedOut:  ``,
 			errorMatcher: IsJSONObjectParse,
 		},
+		{
```
Comment (kopiczko) [inline format_test.go]: In this test `verbosity` has `int` value. Before the change it would fail and print the same JSON.

File: format.go [hunk 2]
```diff
@@ -202,3 +203,20 @@ func getStack(m map[string]string) string {
 
 	return stack
 }
+
+func getString(m map[string]interface{}, key string) string {
```
Comment (tuommaki) [inline format.go]: I remember we had earlier some discussion on `nil` vs. `<nil>` but I can't remember the outcome. This difference made me think here, if following would make more sense:

```go
func getString(m map[string]interface{}, key string) string {
    v, ok := m[key]
    if !ok {
        return "<missing>"
    }

    return fmt.Sprintf("%v", v)
}
```

Behavior on missing entry is always difficult to handle without error, but being explicit on missing value would be more clear for me. That way, `nil` would then become `<nil>` which kinda matches then with `<missing>`. Don't know if this make sense for you. Just something that came up in my mind, but I don't have strong opinion on this. If you want to keep it as-is, I'm fine with that.

Comment (kopiczko) [reply format.go]: What we did recently was `fmt.Sprintf("%v", nil)` which yields `<nil>`. Would that be fine?

Comment (tuommaki) [reply format.go]: Yes.

Comment (kopiczko) [reply format.go]: Okay. Done.

Comment (kopiczko) [reply format.go]: Okay. Done.

**Pull request merged**